rpa-builder/6.7/modules/ROOT/pages/toolbox-google-drive-operations.adoc

```diff
@@ -1,12 +1,15 @@
-= Google Drive Operations
+= Google Workspace
```

Comment:
```suggestion
= Google Workspace Operations
```

Reply:
This matches the name of the container as it shows in the UI